### DISCUSSION TITLE
CONFIG_FIRMWARE_RAM_MOD_256=y

### DISCUSSION THREAD

**Discussion by SharkyEXE** [shvchk/padavan-builder-workflow#84] [Общее]
Всем здравствуйте.

Имею роутер ASUS RT-N56U B1 (перепаяна микросхема/чип оперативной памяти, на выхлопе вместо 128 Мегабайт оперативной памяти раньше стола прошивка, писалось 256 Мегабайт оперативной памяти)

В [build.config] есть строка
CONFIG_FIRMWARE_RAM_MOD_256=y
Собирается прошивка по инструкции
https://github.com/shvchk/padavan-builder-workflow

После установки
пишется
Свободно ОЗУ: | 98.93 MB / 123.52 MB

Специально чтоли в конфиге
https://gitlab.com/hadzhioglu/padavan-ng/-/blob/master/trunk/configs/templates/asus/rt-n56ub1.config
по инструкции
https://github.com/shvchk/padavan-builder-workflow
убрана поддержка у кого 256 Мегабайт оперативной памяти?

В оригинальном конфиге
https://bitbucket.org/padavan/rt-n56u/src/master/trunk/configs/templates/n56ub1_full.config
эта строка есть по дефолту
CONFIG_FIRMWARE_RAM_MOD_256=y

Спасибо.

**Comment by shvchk** [comment]
Это лучше узнавать у мейнтенера [прошивки](https://gitlab.com/hadzhioglu/padavan-ng) (там есть раздел [Issues](https://gitlab.com/hadzhioglu/padavan-ng/-/issues)). Здесь только скрипт сборки :wink:

**Reply by SharkyEXE** [reply]
Понял, спасибо Вам
